src/brackets.js

```diff
@@ -205,6 +206,7 @@ define(function (require, exports, module) {
             ScrollTrackMarkers      : require("search/ScrollTrackMarkers"),
             UpdateNotification      : require("utils/UpdateNotification"),
             WorkingSetView          : WorkingSetView,
+            XMLUtils                : XMLUtils,
```

Comment:
Ah, yes -- that's because of the project-global API `JSUtils.findMatchingFunctions()`, whose unit tests need to boot up a full instance of Brackets with a specific project open.  See JSUtils-test.